front/app/containers/EventsShowPage/components/InformationColumnDesktop.tsx

```diff
@@ -56,13 +56,7 @@ const InformationColumnDesktop = ({ event, className }: Props) => {
                     <EventAttendanceButton event={event} />
                   </Box>
                 )}
-                {event.attributes.attendees_count > 0 && (
```

Comment:
The condition does not relate to the parent component, but rather to the internal component logic (it never makes sense to render this component if the attendee count is 0). If the condition were about viewport width, for example, it would fit here.